packages/blade/codemods/brand-refresh/transformers/index.ts

```diff
@@ -0,0 +1,239 @@
+      (path, index, self) =>
+        (path.node.name.name === 'size' || path.node.name.name === 'variant') &&
+        index === self.findIndex((obj) => path.node.start === obj.node.start),
+    ) // Filter by name `variant` and remove any duplicates
+    .remove();
+
+  // Replace Title with Heading and update the 'size' attribute
+  // <Title size="medium"> to <Heading size="xlarge">
+  typographyJSXElements
+    .filter((path) => path.value.openingElement.name.name === 'Title')
+    // replace with Heading
+    .replaceWith((path) => {
+      const { node } = path;
+
+      node.openingElement.name.name = 'Heading';
+      node.closingElement.name.name = 'Heading';
+
+      const sizeAttribute = node.openingElement.attributes.find(
+        (attribute) => attribute.name.name === 'size',
+      );
+
+      if (isExpression(sizeAttribute)) {
+        console.log(
+          'Expression found in size attribute, please update manually:',
+          `${file.path}:${sizeAttribute.loc.start.line}`,
+        );
+        return node;
+      }
+
+      if (!sizeAttribute) {
+        node.openingElement.attributes.push(
+          j.jsxAttribute(j.jsxIdentifier('size'), j.literal('large')),
+        );
+
+        return node;
+      }
+
+      const otherAttributes = node.openingElement.attributes.filter(
+        (attribute) => attribute.name.name !== 'size',
+      );
+      otherAttributes.push(
+        j.jsxAttribute(
+          j.jsxIdentifier('size'),
+          j.literal(titleToHeadingMap[sizeAttribute.value.value] || 'large'),
+        ),
+      );
+
+      node.openingElement.attributes = otherAttributes;
+
+      return node;
+    });
+
+  // Remove/Update the Title import from "@razorpay/blade/components"
+  root
+    .find(j.ImportDeclaration)
+    .filter((path) => path.value.source.value === '@razorpay/blade/components')
+    .find(j.ImportSpecifier)
+    .filter((path) => path.value.imported.name === 'Title')
+    .replaceWith((path) => {
+      // Check if Heading import is already present
+      const isHeadingImportPresent = path.parent.value.specifiers.some(
+        (node) => node.imported.name === 'Heading',
+      );
+      // // If Heading import is not present, update the "Title" import to use "Heading"
+      if (!isHeadingImportPresent) {
+        path.value.imported.name = 'Heading';
+      } else {
+        // If "Heading" import is present, remove the "Title" import
+        path.parent.value.specifiers = path.parent.value.specifiers.filter(
+          (node) => node.imported.name !== 'Title',
+        );
+      }
+
+      return path.node;
+    });
+
+  // Remove `type` prop from Typography Components
+  typographyJSXElements
+    .find(j.JSXAttribute) // Find all JSX props
+    .filter(
+      (path, index, self) =>
+        path.node.name.name === 'type' &&
+        index === self.findIndex((obj) => path.node.start === obj.node.start),
+    ) // Filter by name `type` and remove any duplicates
+    .remove();
```

Comment:
This we'll handle correctly in colors codemod PR right?

Reply:
yes